src/Components/ArtworkFilter/ArtworkFilterCreateAlert.tsx

```diff
@@ -20,15 +22,43 @@ export const ArtworkFilterCreateAlert: FC<ArtworkFilterCreateAlertProps> = ({
   children,
 }) => {
   const { entity } = useSavedSearchAlertContext()
-  const { aggregations } = useArtworkFilterContext()
+  const { aggregations, filters: rawFilters } = useArtworkFilterContext()
   const filters = usePrepareFiltersForPills()
-
-  // If there is no entity then we don't want to create an alert
-  if (isEmpty(entity)) return null
+  const newAlertModalEnabled = useFeatureFlag("onyx_artwork_alert_modal_v2")
 
   const criteria = getSearchCriteriaFromFilters(entity, filters)
   const metric = filters?.metric ?? DEFAULT_METRIC
 
+  const { artworkAlertComponent, showArtworkAlert } = useArtworkAlert({
+    initialCriteria: rawFilters,
+  })
```

Comment:
**walkthrough**: We're passing in the currently active filters from the `ArtworkFilterContext` as `initialCriteria` here. Once the modal is open, the two sets of attributes begin to drift as a collector adjusts the criteria.